tests/test_csc.py

```diff
@@ -815,6 +816,25 @@ async def test_show_alarms(self):
                 assert data.acknowledgedBy == ""
             assert seen_alarm_names == alarm_names
 
+    @patch("aiohttp.ClientSession.post")
+    async def test_make_log_entry(self, mock_post):
+        """Test the makeLogEntry command."""
+        test_case_json = {"key": "TEST-123", "status": "PASS"}
+        # mock_response = MagicMock()
+        mock_response = AsyncMock()
+        mock_response.json.return_value = test_case_json
+        mock_post.return_value.__aenter__.return_value = mock_response
+
+        # Send the makeLogEntry command.
+        async with self.make_csc(
+            config_dir=TEST_CONFIG_DIR, initial_state=salobj.State.STANDBY
+        ):
+            await salobj.set_summary_state(
+                self.remote, state=salobj.State.ENABLED, override="enabled.yaml"
+            )
+            await self.check_all_alarms_events_are_none()
+            await self.remote.cmd_makeLogEntry.start(timeout=STD_TIMEOUT)
```

Comment:
I think you want to specify an alarm to log and then check that the mock_reponse was called with the values you expected?

Reply:
How do I specify an alarm to log?

Reply:
It would be something like this:

```
await self.remote.cmd_makeLogEntry.set_start(
    name="Enabled. ATDome:0",
    timeout=STD_TIMEOUT
)
```

To expand on the values above.

The test enables the CSC with the `enabled.yaml` configuration, which you can find [here](https://github.com/lsst-ts/ts_watcher/blob/develop/tests/data/config/csc/enabled.yaml).

In there you can see that is configures the "Enabled" [rule](https://github.com/lsst-ts/ts_watcher/blob/eca6e70b842d6e651e2f19edf58e21132f5a4c66/tests/data/config/csc/enabled.yaml#L6) for 3 CSCs, ATCamera, ATDome and ScriptQueue:2. However it disables the rule for ATCamera.

By passing `Enabled. ATDome:0` as the `name` of the alarm to make a log entry for, it means it will log the entry for that alarm in particular. You should be able to force that alarm to be critical by doing this:

```
# Send the makeLogEntry command.
async with self.make_csc(
    config_dir=TEST_CONFIG_DIR, initial_state=salobj.State.STANDBY
), salobj.Controller(name="ATDome", write_only=True) as atdome:

    atdome_alarm_name = "Enabled.ATDome:0"
    await self.check_all_alarms_events_are_none()

    await atdome.evt_summaryState.set_write(
        summaryState=salobj.State.FAULT, force_output=True
    )
    await self.assert_next_alarm(
        name=atdome_alarm_name,
        severity=AlarmSeverity.CRITICAL,
        maxSeverity=AlarmSeverity.CRITICAL,
        acknowledged=False,
        acknowledgedBy="",
    )

    await self.remote.cmd_makeLogEntry.set_start(
        name=atdome_alarm_name,
        timeout=STD_TIMEOUT,
    )

    # assert here that mock_response got the expected payload.

```